x-pack/plugins/infra/public/pages/metrics/hosts/hooks/use_host_query.ts

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
I guess we can remove this file, right?

Reply:
Ah yes, I missed that. Thank you, I removed it ✅